### DISCUSSION TITLE
Sourcecode Linux

### DISCUSSION THREAD

**Discussion by yythoss** [immense/Remotely#230] [General]
Is it possible to provide the source code for the linux server?
I think the Remotely_Server_Linux-x64.zip contains old code.

**Comment by bitbound** [comment]
The source code is here in this repository:  https://github.com/lucent-sea/Remotely/tree/master/Server

The server works on both Linux and Windows.  You just need to publish it for one or the other.

**Comment by yythoss** [comment]
THX

**Comment by yythoss** [comment]
I can not find the section, where I must change the server for linux in the Pulish.ps1

**Comment by yythoss** [comment]
Is this the $RID variable?

**Reply by bitbound** [reply]
Yup.  For reference, this is covered in the readme:

```
* Run Publish.ps1 in the [Utilities folder in source control](https://raw.githubusercontent.com/lucent-sea/Remotely/master/Utilities/Publish.ps1).
    * Example: `powershell -f [path]\Publish.ps1 -outdir C:\inetpub\remotely -rid win10-x64 -hostname https://mysite.mydomain.com`
    	* Use `-rid linux-x64` if you're going to host on Ubuntu.
    * The output folder will now contain the server, with the clients in the Downloads folder.
	* The above hostname will be hardcoded in the screen-sharing desktop apps and the installer.
```

>    	* Use `-rid linux-x64` if you're going to host on Ubuntu.

A full list of RIDs can be found at the link in the Publish.ps1 file.
